EIPS/eip-7514.md

```diff
@@ -42,6 +42,8 @@ This requires no changes to the Execution Layer.
 
 - Modify `get_validator_churn_limit` function adding an upper bound `MAX_PER_EPOCH_CHURN_LIMIT`
 
+The full specification of the proposed change can be found in [`/_features/eip7514/beacon-chain.md`](https://github.com/ethereum/consensus-specs/blob/a56c4d026ff4fd39cbd77b7b8545eae719613018/specs/_features/eip7514/beacon_chain.md).
```

Comment:
But this is a dynamic link and `MAX_PER_EPOCH_CHURN_LIMIT` is still 12 in this commit...! Maybe add the Deneb link later instead.